src/Util/ComposerAutoloaderFinder.php

```diff
@@ -46,13 +56,15 @@ private function findComposerClassLoader()
 
         foreach ($autoloadFunctions as $autoloader) {
             if (\is_array($autoloader) && isset($autoloader[0]) && \is_object($autoloader[0])) {
-                if ($autoloader[0] instanceof ClassLoader) {
+                if ($autoloader[0] instanceof ClassLoader
```

Comment:
Maybe we should improve the `Composer ClassLoader not found ` message above?

> Could not find a Composer autoloader that autoloads from "{$this->rootNamespace}"

What about PSR-0?

And also, there is still an edge case: I have `App\\` configured as my default, but I actually always generate things using a full class name for some reason - e.g. `MyCompany\Entity\FooEntity`. And, I don't have `App` configured in my autoloader. In that case, I would get the error EVEN if `MyCompany` *is* mapped in a valid way.

Perhaps, to at least help reduce this, we should only apply this extra check that you've added IF we find *multiple* class loaders? If we only find 1, it's safe to use that, even if it doesn't make the root namespace.